EmployeeManagement/Tavisca.EmployeeManagement.Translator/Translator.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.ServiceModel;
+using System.Text;
+using Tavisca.EmployeeManagement.DataContract;
+using Tavisca.WCF.Model;
+                         
+namespace Tavisca.EmployeeManagement.Translator
+{
+    // NOTE: You can use the "Rename" command on the "Refactor" menu to change the class name "Service1" in both code and config file together.
+    public static class Translator 
+    {
+        public static EmployeeModel EmployeeModelGenerator(Employee employee)
+        {
+
+            var employeeModel = EmployeeModelFactory.CreateInstance(employee.Title,employee.FirstName, employee.LastName, employee.Email);
+          
+            /*   employeeModel.FirstName = employee.FirstName;
+               employeeModel.LastName = employee.LastName;
+               employeeModel.Email = employee.Email;
+             */
+            return employeeModel as EmployeeModel;  //........checkpoint for error..........
+        }
+
+        public static RemarkModel RemarkModelGenerator(Remark remark)
+        {
+           var remarkModel = RemarkModelFactory.CreateInstance(remark.Text);
+            //To Do: Wrapping
+          //  return remarkModel;
+           return remarkModel as RemarkModel;
+        }
+
+        public static Employee TranslateToEmployee(EmployeeModel employeeModel)
+        {
+           Employee employeeDataContractObject=new Employee();
+           employeeDataContractObject.Id = employeeModel.Id;
+           employeeDataContractObject.Title = employeeModel.Title;
+           employeeDataContractObject.FirstName = employeeModel.FirstName;
+           employeeDataContractObject.LastName = employeeModel.LastName;
+           employeeDataContractObject.Email = employeeModel.Email;
+           employeeDataContractObject.EmployeeRemark = employeeModel.Remark;
+
+           return employeeDataContractObject;
+        }
+
+        public static List<Employee> TranslateToEmployee(List<EmployeeModel> employeeModelList)
+        {
+            List<Employee> employees = new List<Employee>();
+            for (int i = 0; i < employeeModelList.Count - 1; i++)
+            {
+                Employee employeeDataContractObject = new Employee();
+                employeeDataContractObject.Id = employeeModelList[i].Id;
```

Comment:
You could have called the TranslateToEmployee method implemented in the above function. This is duplicate code.